src/eventshub/vite.config.ts

```diff
@@ -7,6 +7,7 @@ export default defineConfig({
   build: {
     rollupOptions: {
       output: {
+        noConflict: true,
```

Comment:
There is a conflict with the generated javascript and the global namespace. Both this file and GTAG declare a `DL` constant and that is causing the app to no load.

I have disabled GTAG on the demo site but we will need to look into this further.

Reply:
To be clear, I don't believe this to be a fix but I haven't validated it yet.